AppCenterAnalytics/AppCenterAnalytics/Internals/MSAnalytics+Validation.h

```diff
@@ -11,12 +12,19 @@ extern NSString *MSAnalyticsValidationCategory;
 
 @interface MSAnalytics (Validation)
 
+/**
+ * Validate AppCenter log.
+ *
+ * @param log The AppCenter log.
+ *
+ * @return YES if AppCenter log is valid; NO otherwise.
+ */
 - (BOOL)validateLog:(MSLogWithNameAndProperties *)log;
 
 /**
  * Validate event name
  *
- * @return YES if event name is valid, NO otherwise.
+ * @return YES if event name is valid; NO otherwise.
```

Comment:
Comma?

Reply:
At least that's what we're doing elsewhere